apps/demo/src/h5wasm/plugin-utils.ts

```diff
@@ -1,25 +1,30 @@
 // Import compression plugins as static assets (i.e. as URLs)
 // cf. `vite.config.ts` and `src/vite-env.d.ts
+import { Plugin } from '@h5web/h5wasm';
```

Comment:
Introducing an enum so it's easier for consumer applications to be exhaustive and not forget to load/support a plugin.

Reply:
Should it be documented ?

Reply:
Good catch! I've updated the README of `@h5web/h5wasm`.